study-builder/studies/angio/consent.conf

```diff
@@ -116,13 +116,13 @@
           "state": "INCOMPLETE",
           "height": 85,
           "width": 70,
-          "1x": "https://storage.googleapis.com/"${assetsBucketName}"/consent_01_1x.png"
+          "1x": "https://storage.googleapis.com/"${assetsBucketName}"/angio/consent_01_1x.png"
```

Comment:
I'm not sure what's the best way to update these links in the production database, since it's environment specific. Maybe just issuing manual SQL updates after we deploy?

Reply:
As part of this ticket/PR, we need to write the SQL, test it in lower env (with env changed as per env) and add it to release notes so that we don't forget that we need to do this.

If executing the SQL in prod DB is not permitted , we might need to come up with a script which we can execute and delete it later.

Reply:
Originally, I had some SQL that will take care of this. But on second thought, since bucket names are globally unique across all of GCP, I think we can script this into a study patch. Let me try it out.

Reply:
done